src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -723,6 +757,94 @@
         </Setter>
     </Style>
 
+    <Style x:Key="InputStyleWithIcon" TargetType="{x:Type TextBox}">
```

Comment:
What are the change to this style?

Reply:
> What are the changes to this style?

adds an extra space to the right side of the input to draw the Icon that will be displayed if required